stan-dev/math
# Update log1p so it can be vectorized in Stan

#### Submisison Checklist
#### Summary:

This pull request is needed in order to vectorize `log1p` in Stan.
#### Intended Effect:

I added the header file for the vectorized `log1p` to `stan/math/prim/mat.hpp`. I also updated `log1p` because the tests failed for me.
#### How to Verify:

Check that `log1p.hpp` is in `stan/math/prim/mat.hpp`. Check also that the vectorize test works for it.
#### Side Effects:

N/A
#### Documentation:

N/A
#### Reviewer Suggestions:
#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):
Per Michigan's copyright laws, Rayleigh Lei owns the copyrights for this work.

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
